lib/ueberauth/strategy/facebook.ex

```diff
@@ -57,6 +57,20 @@ defmodule Ueberauth.Strategy.Facebook do
     end
   end
 
+  @doc """
+  Handles the callback from app with access_token.
+  """
+  def handle_callback!(%Plug.Conn{params: %{"access_token" => access_token}} = conn) do
```

Comment:
@antonmedv it seems these changes go beyond just changing the image, is this necessary?